bu/CopcSupport.cpp

```diff
@@ -46,11 +46,8 @@ CopcSupport::CopcSupport(const BaseInfo& b) : m_b(b),
     m_header.file_source_id = m_b.fileSourceId;
     m_header.creation.day = m_b.creationDoy;
     m_header.creation.year = m_b.creationYear;
-//    m_b.systemId.copy(m_header.system_identifier, 32);
```

Comment:
I think `strncpy` is better because if the string is shorter than 32, the buffer will get filled out with 0's. But it should be properly initialized so meh.

Reply:
`strncpy` was raising a `stringop-truncation` warning. Since `m_header.system_identifier` is initialized `copy` should work correctly for any string size.